x/pocketcore/keeper.go

```diff
@@ -0,0 +1,29 @@
+package pocketcore
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/bank"
+)
+
+// PocketCoreKeeper maintains the link to storage and exposes getter/setter methods for the various parts of the state machine
+type PocketCoreKeeper struct {
+	AccountKeeper     accountKeeper
```

Comment:
Expose some of these keepers publicly so someone with an instance to a PocketCoreKeeper can access these other subkeepers, no need for 2 different instances.

Reply:
Yes, the capitalization of the variable means it is public, the undercapitalization of the type means that the type is not accessible. Are you wanting the keepers to be accessible without the PocketCoreKeeper?

Reply:
Yeah, right now `accountKeeper` is private to the struct, so what I'm recommending is making it public, so someone with an instance of PocketCoreKeeper can access the `AccountKeeper` instance within it.

Reply:
No accountKeeper is not private! accountKeeper the `type` is private. But you can access the public `instance` by doing PocketCoreKeeper.AccountKeeper

Reply:
Got it!